tests/UI/campaigns/functional/BO/08_design/06_imageSettings/14_checkCategoryImageFormat.ts

```diff
@@ -141,14 +136,12 @@ describe('BO - Design - Image Settings - Check category image format', async ()
     },
     {
       category: categoryDataWEBP,
-      extOriginal: 'jpg',
-      // @todo : https://github.com/PrestaShop/PrestaShop/issues/32408
-      // extOriginal: 'webp',
-      extGenerated: 'webp',
-      extImageType: 'jpg',
+      extOriginal: 'webp',
+      extGenerated: 'jpg',
+      extImageType: 'png',
```

Comment:
It should be `webp` and not `png`.

Reply:
Same as the other one, we check the format inside. Always jpg or png, depending on configuration.

Reply:
No, the original file must keep in the original format and must not be changed.

Reply:
The configuration impact generated files.